soh/src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -973,7 +979,11 @@ void EnGo_GetItem(EnGo* this, GlobalContext* globalCtx) {
 
         yDist = fabsf(this->actor.yDistToPlayer) + 1.0f;
         xzDist = this->actor.xzDistToPlayer + 1.0f;
-        func_8002F434(&this->actor, globalCtx, getItemId, xzDist, yDist);
+        if (!gSaveContext.n64ddFlag) {
```

Comment:
Logical Error: Can it get to this point without setting getItemEntry?

Reply:
Same fix as suggested for fishing should work fine.